arch/arm/core/cortex_m/mpu/arm_mpu_v8_internal.h

```diff
@@ -238,11 +238,11 @@ static inline int _is_enabled_region(u32_t index)
  *
  * MPU regions are configurable, however, some platforms might have fixed-size
  * SAU or IDAU regions. So, even if a buffer is allocated inside a single MPU
- * region, it might be span across multiple SAU/IDAU regions, which will make
- * the TT-based address range check fail.
+ * region, it might span across multiple SAU/IDAU regions, which will make the
+ * TT-based address range check fail.
  *
- * Therefore, the function perform a second check, in case the fast address
- * range check fails, which is based on MPU only.
+ * Therefore, the function performs a second check, which is based on MPU only,
+ * in case the fast address range check fails.
```

Comment:
I think these comment edits belong with the previous doc commit

Reply:
Yeap, my bad, will fix that :)

Reply:
This is now fixed.